app/controllers/katello/api/v2/simple_content_access_controller.rb

```diff
@@ -26,8 +26,10 @@ def status
     api :PUT, "/organizations/:organization_id/simple_content_access/enable",
       N_("Enable simple content access for a manifest")
     param :organization_id, :number, :desc => N_("Organization ID"), :required => true
+    param :auto_create_overrides, :bool, :desc => N_("Automatically create disabled content overrides for custom products which do not have an attached subscription"), :required => false, :default => true
     def enable
-      task = async_task(::Actions::Katello::Organization::SimpleContentAccess::Enable, params[:organization_id])
+      auto_create = params.key?(:auto_create_overrides) ? params[:auto_create_overrides] : true
```

Comment:
```suggestion
      auto_create = params.key?(:auto_create_overrides) ? Foreman::Cast.to_bool(params[:auto_create_overrides]) : true
```

Reply:
Where is this param being passed? Is there some UI change needed to pass this parameter? Or is it hammer only?

Reply:
It's API / CLI-only. The default is true so no UI changes were needed.